packages/backend/src/peripherals/zksync/parseZksyncResponse.ts

```diff
@@ -0,0 +1,10 @@
+import { ZksyncResponse } from './schemas'
+
+export function parseZksyncResponse(value: string): ZksyncResponse {
+  try {
+    const json: unknown = JSON.parse(value)
+    return ZksyncResponse.parse(json)
+  } catch {
+    throw new TypeError('Invalid Zksync response.')
```

Comment:
Afaik `zod` should include in the error what did not fit the schema - would be good to log that info somehow for simpler debugging.

Reply:
We use zod in a few places and that would probably be a smart thing to do, but maybe not in the scope of this PR